src/Akka.Hosting/LoggerConfigBuilder.cs

```diff
@@ -47,6 +47,7 @@ internal LoggerConfigBuilder(AkkaConfigurationBuilder builder)
 
         public DebugOptions? DebugOptions { get; set; }
 
+        [Obsolete("Use the WithDefaultLogMessageFormatter<T> method instead")]
```

Comment:
Marked as `Obsolete` - this is wonky. The generic method with constraints is safer, but I don't want to remove this since we reference it in the documentation and samples. Maybe once v1.6 ships.